code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1150,16 +1153,16 @@
 	L.Dizzy(0)
 	L.Jitter(0)
 
-/datum/reagent/medicine/changelingadrenaline/overdose_process(mob/living/M as mob)
-	M.adjustToxLoss(1, 0)
+/datum/reagent/medicine/changelingadrenaline/overdose_process(mob/living/M as mob, delta_time, times_fired)
+	M.adjustToxLoss(1 * REM * delta_time, 0)
 	..()
 	return TRUE
 
 /datum/reagent/medicine/changelinghaste
 	name = "Changeling Haste"
 	description = "Drastically increases movement speed, but deals toxin damage."
 	color = "#AE151D"
-	metabolization_rate = 1
+	metabolization_rate = 2.5 * REAGENTS_METABOLISM
```

Comment:
?

Reply:
I'm using REAGENTS_METABOLISM to scale metabolization rates.